### PR TITLE
DX-112: Deploy to Azure

### PR DESCRIPTION
**Pull request by asbjornu**
I suppose this can just be closed and the accompanying branch deleted, @NullableInt?

### CODE REVIEW

**Comment by NullableInt** [comment]
Due to DevOps/TechSupport never answering my ticket to get access in Azure, this was never finished.
We want to move to Azure so we can have several versions of the site live at the same time, letting us send out feature branch links to externals to review text before we merge it to master/develop.
But I never got to the end.

**Comment by asbjornu** [comment]
Aha. Let's keep this PR as a visible, textual and readable reminder about where we stand in the matter, then.

**Comment by NullableInt** [comment]
@asbjornu do we want to pick this up again now?

File: .github/workflows/feature.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Feature
+on: 
+  push:
+    branches:
+      - "feature/**"
+      - "develop"
```
Comment (asbjornu) [inline .github/workflows/feature.yml:6]:
```suggestion
    branches: [develop]
```

File: .github/workflows/feature.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Feature
```
Comment (asbjornu) [inline .github/workflows/feature.yml:1]:
```suggestion
name: Deploy to stage
```

File: .github/workflows/master.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+name: Master
```
Comment (asbjornu) [inline .github/workflows/master.yml:1]:
```suggestion
name: Deploy to production
```

**Review by asbjornu — COMMENTED** [review]
> @asbjornu do we want to pick this up again now?

Indeed!

**Review by NullableInt — APPROVED** [review]

**Pull request merged**